CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.5.0-beta  
+ 
+This code is considered production quality, but depends on angular2:  
+^3.0.0-alpha+1. The alpha tag represents the evolving nature of the AngularDart  
+api, not code quality (3.0.0-beta+1 is used in production Google apps).  
+  
+ * Rename library to angular_components.  
+
 ## 0.4.1-beta
```

Comment:
Put a note here – all previous releases were named `angular2_components` – so folks don't wonder where the releases on pub are 😄